res/schema.xml

```diff
@@ -489,4 +489,29 @@ METADATA
       ALTER TABLE library ADD COLUMN color INTEGER;
     </sql>
   </revision>
+  <revision version="32" min_compatible="3">
+    <description>
+      Convert the PredefinedColor ID to the actual RGB value.
+    </description>
+    <sql>
+      <!-- No Color becomes Orange (not in default palette) -->
+      UPDATE cues SET color=0xFF8C00 WHERE color=0;
+      <!-- Red -->
```

Comment:
Are these colors defined in the code somewhere?  Can you add a comment in this XML pointing to where that is?